### DISCUSSION TITLE
Filter Users Listing Screen?

### DISCUSSION THREAD

**Discussion by DGeoWils** [statamic/cms#6092] [Q&A] [answered]
Is there a way to add filters to the Users Listing screen in the control panel? I'd like to be able to filter by Role and a custom field.

Thanks!

**Comment by edalzell** [comment, marked as answer]
Nope, but that's a decent feature request, you can add it here: https://github.com/statamic/ideas/issues/new

**Reply by Gonzalo2683** [reply]
Hello, I have reviewed the list of ideas and I cannot find any related to this new functionality. If it's not already created, I can create it.

The idea is that from a user relationship type field, it can be configured to show only those who have a certain role in the list. There currently seems to be no way to do this, I don't know if there is a way to alter this behavior programmatically.
